helm-go-lib/main.go

```diff
@@ -18,15 +18,16 @@ import (
 	"helm.sh/helm/v3/pkg/cli/values"
 	"helm.sh/helm/v3/pkg/getter"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
-	"k8s.io/client-go/rest"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/rest"
```

Comment:
Doesn't need to be this PR necessarily, but should we consider upgrading helm/k8s library versions in https://github.com/broadinstitute/helm-scala-sdk/blob/master/helm-go-lib/go.mod? The current ones are probably fairly old.

Reply:
done